klarna-on-demand/src/main/java/com/klarna/ondemand/RegistrationSettings.java

```diff
@@ -0,0 +1,24 @@
+package com.klarna.ondemand;
+
+import java.io.Serializable;
+
+/**
+ * Defines the settings for Klarna's Registration Activity
+ */
+public class RegistrationSettings implements Serializable {
+    private String confirmedUserDataId;
+    private String prefillPhoneNumber;
+
+    RegistrationSettings(RegistrationSettingsBuilder builder) {
```

Comment:
Actually both ways are generally OK [(example)](http://www.javacodegeeks.com/2012/07/builder-design-pattern-in-java.html)

Nevertheless, I prefer it the way you suggested, so I'll change it.